docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -155,27 +155,46 @@ for search. Many searches will need to retrieve only a small subset of the
 total shard data before returning results.
 
 To mount a searchable snapshot index with the shared cache mount option, you
-must configure the `xpack.searchable.snapshot.shared_cache.size` setting to
-reserve space for the cache on one or more nodes. Indices mounted with the
-shared cache mount option are only allocated to nodes that have this setting
-configured.
+must have one or more nodes with a shared cache available. By default,
+dedicated frozen data tier nodes (nodes with the `data_frozen` role and no other
+data roles) have a shared cache configured using the greater of 90% of total
+disk space and total disk space subtracted a headroom of 100GB.
```

Comment:
I wonder if inserting an example here would help.

Reply:
Good idea, added examples in 53f5982f87614c09cb86d561d0cee676afd8ff84